fetch.bs

```diff
@@ -4156,6 +4156,33 @@ these steps:
    <li><p>Let <var>requestForServiceWorker</var> be a <a for=request>clone</a> of
    <var>request</var>.
 
+   <li>
+    <p> If <var>requestForServiceWorker</var>'s <a for=/>body</a> is non-null, then:
+
+    <ol>
+     <li> Let <var>transformAlgorithm</var> given <var>chunk</var> be these steps:
+      <ol>
+        <li><p>If the ongoing fetch is <a for=fetch>terminated</a>, then abort these steps.
+
+        <li><p>If <var>chunk</var> is not a {{Uint8Array}} object, then
+         <a lt=terminated for=fetch>terminate</a> the ongoing fetch.
+
+        <li>Otherwise, <a for=ReadableStream>enqueue</a> <var>chunk</var>. The user agent may split
+         the chunk in <a>implementation-defined</a> practical size and
+         <a for=ReadableStream>enqueues</a> each of them.
+      </ol>
+
+     <li> Let <var>transformStream</var> be result of <a for=TransformStream>setting up</a> a
+      {{TransformStream}} with <var>transformAlgorithm</var>.
+
+     <li> Perform <a href=https://streams.spec.whatwg.org/#readable-stream-pipe-to>ReadableStreamPipeTo</a>(
```

Comment:
If you wait a little longer we can land https://github.com/whatwg/streams/pull/1113 which will make this cleaner.

Reply:
Applied the link.